content/intro-to-storybook/ember/en/test.md

```diff
@@ -26,86 +26,78 @@ Visual regression tests are designed to catch changes in appearance. They work b
 
 Storybook is a fantastic tool for visual regression testing because every story is essentially a test specification. Each time we write or update a story we get a spec for free!
 
-There are a number of tools for visual regression testing. For professional teams we recommend [**Chromatic**](https://www.chromatic.com/), an addon made by Storybook maintainers that runs tests in the cloud.
+There are a number of tools for visual regression testing. We recommend [**Chromatic**](https://www.chromatic.com/), a free publishing service made by the Storybook maintainers that runs visual tests in parallelized cloud. It also allows us to publish Storybook online as we saw in the [previous chapter](/ember/en/deploy/).
 
-## Setup visual regression testing
+## Catch a UI change
 
-Chromatic is a hassle-free Storybook addon for visual regression testing and review in the cloud. Since it’s a paid service (with a free trial), it may not be for everyone. However, Chromatic is an instructive example of a production visual testing workflow that we'll try out for free. Let’s have a look.
+Visual regression testing relies on comparing images of the new rendered UI code to the baseline images. If a UI change is caught we'll get notified.
 
-### Bring git up to date
+Let's see how it works by tweaking the background of the `Task` component.
 
-Ember CLI has created a git repo for your project; let's check in the changes we have made:
+Start by creating a new branch for this change:
 
 ```bash
-$ git add -A
+git checkout -b change-task-background
 ```
 
-Now commit the files.
-
-```bash
-$ git commit -m "taskbox UI"
+Change `Task` to the following:
+
+```hbs
+<!-- app/components/task.js -->
```

Comment:
Not quite sure about this code here. Doesn't look like turning this into red bg?!?

Reply:
@gossi you're right i copied over the wrong snippet. I'll update it tomorrow.